<ui version="4.0" >
 <class>generalPage</class>
 <widget class="QWidget" name="generalPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>537</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="issueIcon" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string comment="KDE::DoNotExtract" >TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="issueText" >
       <property name="text" >
        <string>&lt;b>There are some issues in your configuration. Please check the Capabilities page for more details.&lt;/b></string>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>738</width>
        <height>433</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Settings and Profiles</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QGroupBox" name="GeneralSettingsBox" >
         <property name="title" >
          <string>General Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout_6" >
          <property name="fieldGrowthPolicy" >
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="lockScreenOnResume" >
            <property name="toolTip" >
             <string>Locks screen when waking up from suspension</string>
            </property>
            <property name="statusTip" >
             <string/>
            </property>
            <property name="whatsThis" >
             <string>You will be asked for a password when resuming from sleep state</string>
            </property>
            <property name="text" >
             <string>Lock screen on resume</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="enableDPMSBox" >
            <property name="text" >
             <string>Let PowerDevil manage screen powersaving</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="notificationsBox" >
            <property name="toolTip" >
             <string>Enables standard notifications</string>
            </property>
            <property name="statusTip" >
             <string/>
            </property>
            <property name="whatsThis" >
             <string>Notifications such as standard events or profile change will be shown</string>
            </property>
            <property name="text" >
             <string>Enable notifications</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QCheckBox" name="warningNotificationsBox" >
            <property name="toolTip" >
             <string>Enables warning notifications</string>
            </property>
            <property name="whatsThis" >
             <string>Notifications such as warnings or important events will be shown. It is recommended to leave this enabled.</string>
            </property>
            <property name="text" >
             <string>Enable warning notifications</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QPushButton" name="notificationsButton" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Configure Notifications...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="4" column="0" >
           <widget class="QCheckBox" name="suspendWait" >
            <property name="toolTip" >
             <string>Interval for interactive confirm notifications</string>
            </property>
            <property name="whatsThis" >
             <string>If selected, before doing an automatic suspension PowerDevil will show an interactive notification for the specified time. If that notification is then clicked, the queued action will not be run.</string>
            </property>
            <property name="text" >
             <string>Before doing a suspend action, wait</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="KIntSpinBox" name="suspendWaitTime" >
            <property name="maximumSize" >
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="suffix" >
             <string comment="As in &quot;seconds&quot;" > sec</string>
            </property>
            <property name="maximum" >
             <number>300</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Profile Assignment</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>When AC Adaptor is plugged in</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="KComboBox" name="acProfile" >
            <property name="maximumSize" >
             <size>
              <width>280</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>When on battery power</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KComboBox" name="batteryProfile" >
            <property name="maximumSize" >
             <size>
              <width>280</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>When battery is at low level</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="KComboBox" name="lowProfile" >
            <property name="maximumSize" >
             <size>
              <width>280</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_11" >
            <property name="text" >
             <string>When battery is at warning level</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="KComboBox" name="warningProfile" >
            <property name="maximumSize" >
             <size>
              <width>280</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>459</width>
        <height>253</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Advanced Battery Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Battery Levels</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2" >
          <property name="fieldGrowthPolicy" >
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Battery is at low level at</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="KIntSpinBox" name="lowSpin" >
            <property name="maximumSize" >
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Low battery level</string>
            </property>
            <property name="whatsThis" >
             <string>Battery will be considered low when it reaches this level</string>
            </property>
            <property name="suffix" >
             <string>%</string>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Battery is at warning level at</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KIntSpinBox" name="warningSpin" >
            <property name="maximumSize" >
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Warning battery level</string>
            </property>
            <property name="whatsThis" >
             <string>Battery will be considered at warning level when it reaches this level</string>
            </property>
            <property name="suffix" >
             <string>%</string>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Battery is at critical level at</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="KIntSpinBox" name="criticalSpin" >
            <property name="maximumSize" >
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Critical battery level</string>
            </property>
            <property name="whatsThis" >
             <string>Battery will be considered critical when it reaches this level</string>
            </property>
            <property name="suffix" >
             <string>%</string>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Battery Actions</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3" >
          <property name="fieldGrowthPolicy" >
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="BatteryCriticalLabel" >
            <property name="text" >
             <string>When battery remaining is critical</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>BatteryCriticalCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="KComboBox" name="BatteryCriticalCombo" >
            <property name="maximumSize" >
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
